app/serializers/event_serializer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class EventSerializer
+class EventSerializer < ApplicationSerializer # ~> NameError: uninitialized constant ApplicationSerializer
```

Comment:
What's this error message?

Reply:
@bibliotechy - That's from the Seeing Is Believing Vim plugin to execute ruby from within Vim.  That was generated by an errant keystroke. Please flag them if you see them. They should not be in the code.